src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/WinForms/IMyAppXamlFileAccessor.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.WindowsForms;
+
+/// <summary>
+/// Provides access to properties in the myapp file.
+/// </summary>
+internal interface IMyAppXamlFileAccessor
```

Comment:
As it turns out this isn't a XAML file, could this just be `IMyAppFileAccessor`?